http2_general/test_h2_frame.py

```diff
@@ -46,20 +49,203 @@ def test_empty_last_data_frame(self):
 
         self.__assert_test(client=deproxy_cl, request_body=request_body, request_number=1)
 
+    def test_empty_data_frame_mixed(self):
+        """
+        Send request with empty DATA frame after non empty.
+        Tempesta treats empty DATA frames without END_STREAM as inavalid.
+        """
+        self.start_all_services()
+        deproxy_cl = self.get_client("deproxy")
+        deproxy_cl.parsing = False
+        request_body = "123"
+
+        deproxy_cl.update_initial_settings()
+        deproxy_cl.send_bytes(deproxy_cl.h2_connection.data_to_send())
+        deproxy_cl.wait_for_ack_settings()
+
+        deproxy_cl.make_request(request=self.post_request, end_stream=False)
+        deproxy_cl.make_request(request=request_body, end_stream=False)
+        deproxy_cl.send_bytes(DataFrame(stream_id=1, data=b"").serialize(), expect_response=False)
+        deproxy_cl.send_bytes(DataFrame(stream_id=1, data=b"").serialize(), expect_response=False)
```

Comment:
You will receive GOAWAY to the first empty DATA frame without END_STREAM. I think this line is not needed.